src/components/priceDisplay.js

```diff
@@ -26,27 +26,25 @@ export const PriceDisplay = (props) => {
             size={props.size}
             display="inline-block"
           >
-            <sup>
-              <div className="dollarSign">$</div>
-            </sup>
+            <Text fontSize="20px" verticalAlign="top" display="inline-block">
```

Comment:
Long story short, the way that we aligned these pieces of text with the top of the price, superscripting it then doing a Ytransform, wound up being buggy with my implementation. So I implemented these instead with `vertical-align: top` styling. Not quite perfect match to the mocks but it fixed it and is a bit cleaner in the styling.

Reply:
Sounds good